src/_utils/icon/index.tsx

```diff
@@ -14,6 +14,11 @@ const StyledBaseIcon = styled(BaseIcon)`
     box-shadow: ${shadow.icon};
   }
 
+  &.kirk-icon--disabled,
+  &.kirk-icon--disabled * {
+    stroke: ${color.fadedText};
```

Comment:
I would rather use styled-components approach and do not apply a class for change the colors based on props

```suggestion
    stroke: ${props => (props.isDisabled ? color.gray : '')};
```

in that case, you only need to pass the property `isDisabled` to the element no need to recalc any class

Reply:
Done